azkaban-common/src/main/java/azkaban/imagemgmt/permission/PermissionManagerImpl.java

```diff
@@ -121,4 +117,29 @@ public boolean isAzkabanAdmin(final User user) {
     return user.getRoles().stream()
         .anyMatch(role -> userManager.getRole(role).getPermission().isPermissionSet(Type.ADMIN));
   }
+
+  /**
+   * Validate each userId's identity using userManager. userId can be group or individual user account.
+   *
+   * @param ids
+   * @throws ImageMgmtValidationException
+   */
+  @Override
+  public void validateIdentity(List<String> ids) {
+    Set<String> invalids = ids.stream()
+        .filter(id -> !userManager.validateGroup(id) && !userManager.validateUser(id))
```

Comment:
IIRC, the group can be validated by "validateGroup" here is merely a subset of valid groups used in validateUserGroupMembership

Reply:
let's have an offline discussion since this is Linkedin specific.